Bottom: 2ca74dcf2839db0641d982acca9ca6f777c3dda2
Top:    51dcc6f39fb363d23d1f84a816ba600c7016c9ff
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 12:42:44 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index c532d8a..70ace2e 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -24,7 +24,7 @@ config SND_SOC_MPC8610_HPCD
 
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
-	depends on SND_MPC52xx_SOC && PPC_BESTCOMM
+	depends on PPC_MPC52xx && PPC_BESTCOMM
 	select SND_SOC_OF_SIMPLE
 	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
@@ -33,7 +33,7 @@ config SND_SOC_MPC5200_I2S
 
 config SND_SOC_MPC5200_AC97
 	tristate "Freescale MPC5200 PSC in AC97 mode driver"
-	depends on SND_MPC52xx_SOC && PPC_BESTCOMM
+	depends on PPC_MPC52xx && PPC_BESTCOMM
 	select SND_SOC_OF_SIMPLE
 	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
